app/views/users/sessions/login_email_sent.html.erb

```diff
@@ -1,18 +1,17 @@
 <% content_for :title, "Check email" %>
```

Comment:
This should match the h1 - worth setting a variable and using it in both places maybe? (or use the locale files?)
```suggestion
<% content_for :title, "Check your email" %>
```